crates/executor/src/execution_state.rs

```diff
@@ -29,19 +29,28 @@ mod versioned_constants {
     const BLOCKIFIER_VERSIONED_CONSTANTS_JSON_0_13_0: &[u8] =
         include_bytes!("../resources/versioned_constants_13_0.json");
 
+    const BLOCKIFIER_VERSIONED_CONSTANTS_JSON_0_13_1: &[u8] =
+        include_bytes!("../resources/versioned_constants_13_1.json");
+
+    const STARKNET_VERSION_0_13_1: StarknetVersion = StarknetVersion::new(0, 13, 1, 0);
+
     const STARKNET_VERSION_MATCHING_LATEST_BLOCKIFIER: StarknetVersion =
-        StarknetVersion::new(0, 13, 1, 0);
+        StarknetVersion::new(0, 13, 1, 1);
 
     lazy_static::lazy_static! {
         pub static ref BLOCKIFIER_VERSIONED_CONSTANTS_0_13_0: VersionedConstants =
             serde_json::from_slice(BLOCKIFIER_VERSIONED_CONSTANTS_JSON_0_13_0).unwrap();
+
+        pub static ref BLOCKIFIER_VERSIONED_CONSTANTS_0_13_1: VersionedConstants =
+            serde_json::from_slice(BLOCKIFIER_VERSIONED_CONSTANTS_JSON_0_13_1).unwrap();
     }
 
     pub(super) fn for_version(version: &StarknetVersion) -> &'static VersionedConstants {
-        // Right now we only properly support two versions: 0.13.0 and 0.13.1.
         // We use 0.13.0 for all blocks _before_ 0.13.1.
-        if version < &STARKNET_VERSION_MATCHING_LATEST_BLOCKIFIER {
+        if version < &STARKNET_VERSION_0_13_1 {
             &BLOCKIFIER_VERSIONED_CONSTANTS_0_13_0
+        } else if version < &STARKNET_VERSION_MATCHING_LATEST_BLOCKIFIER {
+            &BLOCKIFIER_VERSIONED_CONSTANTS_0_13_1
```

Comment:
Minor nit: should we just follow the same convention here?

```suggestion
        } else if version < &STARKNET_VERSION_0_13_1_1 {
            &BLOCKIFIER_VERSIONED_CONSTANTS_0_13_1
```

Reply:
Done.